test/functional/rpc_rawtransaction.py

```diff
@@ -178,6 +182,67 @@ def getrawtransaction_tests(self):
         block = self.nodes[0].getblock(self.nodes[0].getblockhash(0))
         assert_raises_rpc_error(-5, "The genesis block coinbase is not considered an ordinary transaction", self.nodes[0].getrawtransaction, block['merkleroot'])
 
+    def getrawtransaction_verbosity_tests(self):
+        tx = self.nodes[2].sendtoaddress(self.nodes[1].getnewaddress(), 1)
+        [block1] = self.generate(self.nodes[2], 1)
+        fields = [
+            'blockhash',
+            'blocktime',
+            'confirmations',
+            'hash',
+            'hex',
+            'in_active_chain',
+            'locktime',
+            'size',
+            'time',
+            'txid',
+            'vin',
+            'vout',
+            'vsize',
+            'weight',
+        ]
+        prevout_fields = [
+            'generated',
+            'height',
+            'value',
+            'scriptPubKey',
+        ]
+        script_pub_key_fields = [
+            'address',
+            'asm',
+            'hex',
+            'type',
+        ]
+        for n in [0, 3]:
+            for v in [1,2]:
```

Comment:
```diff
 from collections import OrderedDict
 from decimal import Decimal
+from itertools import product
 
@@ -213,30 +214,29 @@ class RawTransactionsTest(BitcoinTestFramework):
-        for n in [0, 3]:
-            for v in [1,2]:
+        for n, v in product([0, 3], [1, 2]):
```
(and reduce indentation in the for-loop scope by four spaces)

Reply:
Change. I also added a comment to indicate intent of the loop.